README.md

```diff
@@ -28,6 +28,13 @@ Select configuration options for page derivatives, Parent Solr Field, and select
 
 Further documentation for this module is available at [our wiki](https://wiki.duraspace.org/display/ISLANDORA/Book+Solution+Pack).
 
+## Known Issues
```

Comment:
I think I would make this a sub-sub-section (`### Known Issues`) at the end of `## Troubleshooting/Issues` to be consistent with the precedence set in [`islandora_xml_forms/README.md`](https://github.com/Islandora/islandora_xml_forms/blob/7.x/README.md#known-issues). Also, I would suggest to either not break lines within sentences, or to break at 80 characters.